python/cudf/cudf/tests/test_csv.py

```diff
@@ -766,6 +766,43 @@ def test_csv_reader_bools(tmpdir, names, dtypes, data, trues, falses):
     assert_eq(df_out, out)
 
 
+def test_csv_reader_bools_NA():
```

Comment:
Pandas csv reader has `false_values` support, can we not write this pytest by comparing against pandas's csv reader output rather than hand-coding the comparison?

Reply:
Unknown values `"qux"` returns null in our reader, but pandas fails to parse. That's is why comparison against pandas was not done in this test.

Reply:
In that case can we have two separate tests? One validating against pandas and one validating for `qux`?

Reply:
Testing against pandas / splitting this test would be great.

Reply:
done. There are differences between our parses and pandas.